domain-logic/stories/__tests__/markStoryDevelopment.test.ts

```diff
@@ -0,0 +1,42 @@
+import { stories } from 'stories'
+import { getPrisma, clearDatabase } from 'db'
+
+const markStoryDevelopment = stories.markStoryDevelopment
```

Comment:
Nothing really important, just acting as a styling linter here =P
```suggestion
const { markStoryDevelopment } = stories
```